www/_posts/2016-07-27-cordova-windows-4.4.2.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+author:
+    name: Vladimir Kotikov
+    url: https://github.com/vladimir-kotikov
+title:  "Apache Cordova Windows 4.4.2"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that `Cordova Windows 4.4.2` has been released!
+
+This release fixes some issues we've missed in 4.4.1. In particular, we've completely fixed all build issues with
+new VS build tools. For the rest of changes see release notes below.
```

Comment:
I'm wondering how best to express this. Maybe something like 'In particular, we have fixed build issues experienced with the new install experience in Visual Studio "15" previews'. Do you have any thoughts @mbraude?